Added pyshark and tshark to python.yaml rosdep rules

jack-digilabs opened the pull request:
Added pyshark and tshark to the rosdep rules

tfoote commented on rosdep/python.yaml:
```diff
@@ -2656,6 +2656,10 @@ python-pyramid:
   fedora: [python-pyramid]
   gentoo: [dev-python/pyramid]
   ubuntu: [python-pyramid]
+python-pyshark:
```
If this is pip only it should have a `-pip` ending.

tfoote commented on rosdep/python.yaml:
```diff
@@ -3893,6 +3897,8 @@ python-trep:
     pip:
       packages: [trep]
+python-tshark:
+  ubuntu: [tshark]
```
Please contribute at least debian rules too. Fedora macOS and Gentoo would be appreciated as well.

jack-digilabs commented:
I added Debian to tshark but I'm not certain about any platforms for tshark. I added Debian, fedora, gentoo, and osx to the pyshark rule and the -pip suffix.

tfoote commented on rosdep/python.yaml:
```diff
@@ -2656,6 +2656,22 @@ python-pyramid:
+      packages: [pyshark]
+  osx:
+    pip:
+      packages: [pyquery]
```
This is surprising to have a different pip name on osx. Can you link to a resource that explains why this is different?

tfoote commented on rosdep/python.yaml:
```diff
@@ -2656,6 +2656,22 @@ python-pyramid:
+      packages: [pyshark]
+  ubuntu:
+    pip:
+      packages: [pyshark]
```
Since these are pip wrappers for the underlying too. I suspect that they need tshark installed to be useful. If so these entries should have a depend key as well onto tshark.

mikaelarguedas commented:
@jack-digilabs friendly :bellhop_bell:

dirk-thomas commented:
@jack-digilabs  Closing due to no response.
